markdown/ml/ml8_testing/index.md

```diff
@@ -4,25 +4,75 @@ title: "NN8 - Performanzanalyse"
 menuTitle: "NN8 - Performanzanalyse"
 author: "Canan Yıldız (Türkisch-Deutsche Universität)"
 weight: 8
-outcomes: false
+outcomes: true
 # readings:
 #   - key: "Russell2020"
 #     comment: "Kapitel 2 und 3"
 #   - key: "Ertel2017"
 # quizzes:
 #   - link: XYZ
 #     name: "Testquizz (URL from 'Invite more Players')"
-# assignments:
-#   - topic: blatt01
-#   - topic: blatt02
-# youtube:
-#   - id: XYZ (ID)
-#   - id: XYZ (ID)
-sketch: true
+youtube:
+    - id: T-WYL28iwdU
+    - id: fpsNzn4Moow
+    - id: Wx_HAuIXTAQ
+    - id: UAV7EpdIe6Q
+    - id: vsmoYiArtrA
+sketch: false
 ---
 
 
-<div class="sketch" style="text-align:center;">
-Inhalt befindet sich im Aufbau<br>
-und wird rechtzeitig bereitgetellt.
-</div>
+## Kurze Übersicht
+
+### Performanzmetriken für Klassifizierungsprobleme
+
+#### Wahrheitsmartix (engl. Confusion Matrix)
+*   Gibt eine Übersicht über die Anzahl von richtig und falsch klassifizierten Datenpunkten
```

Comment:
Frage: Die Aussagen beziehen sich auf ein 2-Klassenproblem mit den Klassen 0 und 1. Sollte hier noch eine entsprechende Anmerkung hin oder hattest Du in der Vorlesung immer genau diese beiden Klassen genutzt?

Reply:
Genau, wir haben uns hauptsächlich auf binäre Klassifizierungsprobleme beschränkt. Aber Du hast recht, man sollte zumindest dazu bemerken, dass diese Definition für den Spezialfall gilt.
Ich habe jetzt einen Hinweis hinzugefügt (auch in die Image-Caption), und noch einen zusätzlichen Punkt mit der allgemeineren Definition.

Reply:
passt 👍